lib/ansible/modules/cloud/azure/azure_rm_webapp.py

```diff
@@ -137,12 +145,12 @@
 
     dns_registration:
         description:
-            - If true web app hostname is not registered with DNS on creation.
+            - If set C(true) web app hostname is not registered with DNS on creation.
```

Comment:
Is this description accurate? If I set `dns_registration=true`, the web app hostname is NOT registered? The name of the parameter makes it sound like the opposite. I've kept the meaning the same in my suggestion below, but I think this may be a mistake.
```suggestion
            - Whether or not the web app hostname is registered with DNS on creation. Set to C(false) to register.
```